### ISSUE TITLE
Can't support utf-8?

### ISSUE THREAD

**Issue by elechuanwu** [timofurrer/colorful#6]
Using colorful can not print Chinese characters. This module supports ASCII only?

**Comment by timofurrer** [comment]
Can you provide an example which reproduces your issue?

**Comment by elechuanwu** [comment]
![image](https://user-images.githubusercontent.com/22742613/29016190-f4fc6a16-7b83-11e7-968d-9b76248f3a00.png)

**Comment by timofurrer** [comment]
Seems to be a problem.
I'll have a look at it as soon as I have some time.

In the meantime to can explicitly convert it to unicode before printing:

```python
print(unicode(colorful.red(u'🐧🎉🐧')))
```

When using Python 3 it won't be a problem:

```python
print(colorful.red('🐧🎉🐧'))
print(colorful.red('漢字'))
```

**Comment by Agnewee** [comment]
Chinese is supported now

**Comment by timofurrer** [comment]
Fixed with #9

**Comment by elechuanwu** [comment]
Thank you.
